packages/terra-notification-dialog/translations/sv-SE.json

```diff
@@ -1,6 +1,6 @@
 {
-  "Terra.notification.dialog.hazard-low": "Information",
-  "Terra.notification.dialog.error": "Fel",
-  "Terra.notification.dialog.hazard-medium": "Varning",
-  "Terra.notification.dialog.hazard-high": "Avisering"
+  "Terra.notification.dialog.hazard-low":	"Information",
+  "Terra.notification.dialog.error":	"Fel",
+  "Terra.notification.dialog.hazard-medium":	"Obs!",
+  "Terra.notification.dialog.hazard-high":	"Varning"
```

Comment:
It seems like extra white space was added to the strings.

Reply:
Good catch, removed extra spacing in https://github.com/cerner/terra-framework/pull/2136/commits/3b3f5cb89b3b6b8fc984399a530c3ac5bc9bb7d7